server/coordinator/procedure/transfer_leader.go

```diff
@@ -0,0 +1,225 @@
+package procedure
+
+import (
+	"context"
+	"github.com/CeresDB/ceresdbproto/pkg/clusterpb"
+	"github.com/CeresDB/ceresmeta/server/cluster"
+	"github.com/CeresDB/ceresmeta/server/schedule"
+	"github.com/looplab/fsm"
+	"github.com/pkg/errors"
+)
+
+const (
+	EventTransferLeaderPrepare = "EventTransferLeaderPrepare"
```

Comment:
`fsm.Event` definition
```
func (f *FSM) Event(event string, args ...interface{}) error 
```
Because of the definition of this function, we cannot use iota instead of string